Makefile

```diff
@@ -16,7 +16,7 @@ buf: clean
 	rm -rf src/viam/gen
 	chmod +x plugin/main.py
 	uv pip install protoletariat
-	uv pip install protobuf --upgrade
+	uv pip install protobuf==5.29.1
```

Comment:
@njooma this was the one part I wasn't sure on from your commit, I went with the hard locking to `5.29.1` but will change back if that's not right!

Reply:
:+1: I like pinning the version used! That way, when a new employee joins the team in 6 months and gets protobuf 5.30.0, you don't need to track down bugs that happen on your machine but not on theirs, or vice versa.

Slight preference towards pinning the proletariat version (and all other dependencies), too, but that's perhaps off-topic for the current PR.